### DISCUSSION TITLE
Very Slow upload issue

### DISCUSSION THREAD

**Discussion by nhAsif** [ROM-builders/temporary#18650] [General]
upload speed has become very slow @Apon77 please fix this
https://cirrus-ci.com/task/5017806752710656?logs=upload#L1
https://cirrus-ci.com/task/6112894631280640?logs=upload#L1

**Comment by Apon77** [comment]
how?

**Reply by nhAsif** [reply]
we can use file.io or wetransfer with help of this repo https://github.com/Mikubill/transfer .. these have fast upload speed

curl -sL https://git.io/file-transfer | sh && ./transfer wet out/target/product/$device/$romname
